### ISSUE TITLE
Diacritics

### ISSUE THREAD

**Issue by mtrx84** [TarekRaafat/autoComplete.js#377]
Hi. I am using diacritical mark support. From the Polish letters "ą, ć, ę, ł, ń, ó, ś, ź, ż" I have a problem with the character - "ł", "Ł". It doesn't work when I use "l", "L" interchangeably. Can I fix it somehow?

**Comment by folknor** [comment]
"ł" and "Ł" are not diacritics. They are normal letters.

However, https://www.npmjs.com/package/diacritic will fix it for you. Just implement your own search function.

**Comment by jtauber** [comment]
Yes, just to add more explanation: the diacritics: true just strips unicode combining characters in the range U+0300 to U+036F.

**Comment by mtrx84** [comment]
Thanks both :)
